apps/www/components/Article/Page.js

```diff
@@ -387,6 +391,8 @@ const ArticlePage = ({
     }
   }, [trialSignup])
 
+  const showPlayButton = !extract && hasAccess && isAudioQueueAvailable
```

Comment:
What's the reason for not showing the play-button to non-members?

Reply:
Currently the button doesn't work for logged-out users. (I described why above in the PR)